src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Delivery.java

```diff
@@ -44,6 +44,14 @@
         public String getTranslationKey() {
             return translationKey;
         }
+        
+        /**
+         * Get the lowercase name for freemarker.
+         * @return {@link Enum#name() name()}{@link String#toLowerCase() .toLowerCase()}
+         */
+        public String lowerCaseName() {
```

Comment:
If you want to render the *name* you should use it's translation key and run that through `i18n.translate(state.translationKey)`.